packages/synapse-react-client/src/lib/containers/dataaccess/AccessRequirementDashboard.tsx

```diff
@@ -139,52 +142,60 @@ export function AccessRequirementDashboard(
       />
       <div className="InputPanel">
         <div>
-          <FormLabel htmlFor="ar-name-filter">
-            Filter by Access Requirement Name
-          </FormLabel>
-
-          <InputGroup>
-            <FormControl
-              id="ar-name-filter"
-              type="text"
-              placeholder="Search for an Access Requirement Name"
-              value={nameContains}
-              onChange={e => {
-                setNameContains(e.target.value)
-              }}
-            />
-            <IconSvg icon="searchOutlined" />
-          </InputGroup>
-        </div>
-        <div>
-          <FormLabel htmlFor="project-id-filter">Filter by Project</FormLabel>
-          <div className="ProjectIdInputGroup">
-            <FormControl
-              id="project-id-filter"
-              type="text"
-              placeholder="Enter a project SynID"
-              value={relatedProjectId}
-              onChange={e => {
-                const newValue = e.target.value
-                if (newValue === '') {
-                  setRelatedProjectId(undefined)
-                } else {
-                  setRelatedProjectId(newValue)
-                }
-              }}
-            />
-            <Button
-              variant="outline"
-              onClick={() => {
-                setShowEntityFinder(true)
-              }}
-            >
-              Browse
-            </Button>
-          </div>
+          <TextField
+            label="Filter by Access Requirement Name"
+            id="ar-name-filter"
```

Comment:
main: 
<img width="1472" alt="main_AccessRequirementDashboard" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/ce7779a6-b90e-4b7c-b235-40747da3c408">

feature: 
<img width="1457" alt="feature_AccessRequirementDashboard" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/3608ef65-db40-4822-a8ec-9300ed154646">

Reply:
Button height no longer matches input height

Reply:
True! Fixed in the next commit

Reply:
<img width="1169" alt="feature_AccessRequirementDashboard_buttonHeight" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/73381ff8-bdcd-43fe-900d-a91b3761a9bd">